install/installer/pkg/components/spicedb/data/schema.yaml

```diff
@@ -33,6 +33,10 @@ schema: |-
     // Only owners can change Git providers
     permission read_git_provider = owner + member
     permission write_git_provider = owner
+
+    // Only owners can create and modify billing information
+    permission read_billing = member + owner
```

Comment:
Hm. Do we really have members being able to see anything related to billing? Just came across this exampple, which I think only owners are able to do today: findStripeSubscriptionId ([source](https://github.com/gitpod-io/gitpod/pull/18056/files#diff-d6fdb9bb15400f4fbba00db81b2ce6cb7f7bb566dd97b93b9fd4e3fc9a79f479R4180))

WDYT? :thinking:

Reply:
Yes, the current model does allow them to invoke that method as far as I can tell. But maybe I'm interpreting the `get` permission on CostCenter wrong

Reply:
I think you are right.